frontend/src/components/topbar.tsx

```diff
@@ -41,6 +41,16 @@ import {
 import { Badge } from "@ui/badge";
 import { TopbarAlerts } from "./alert/topbar";
 
+const Docs = () => (<a
```

Comment:
Can you move this block of code below the `Topbar` function, above the `Version` function?